__test__/creds/VerifiableCredential.test.js

```diff
@@ -138,20 +138,112 @@ describe('Unit tests for Verifiable Credentials', () => {
     expect(filtered.claim.identity.name.familyNames).not.toBeDefined();
   });
 
-  // TODO enable me when CCS-514 is done
-  test.skip('Filter claims of composite UCA', () => {
-    const phoneNumber = {
-      country: 'BRZ',
-      countryCode: '+55',
-      number: '31995383635',
-      lineType: 'mobile',
+  it('Should filter claims for Email asking for cvc:Contact:email and return them on the filtered VC', () => {
+    const email = {
+      domain: {
+        tld: 'oVaPsceZ4C',
+        name: 'UTpHKFyaaB',
+      },
+      username: 'ZcMpCBQ0lE',
+    };
+
+    const emailUca = new UCA('cvc:Contact:email', email, '1');
+    const emailCredential = new VC('cvc:Credential:Email', '', null, [emailUca], '1');
+    const filtered = emailCredential.filter(['cvc:Contact:email']);
+
+    expect(filtered.claim.contact.email.domain).toBeDefined();
+    expect(filtered.claim.contact.email.domain.tld).toBe('oVaPsceZ4C');
+    expect(filtered.claim.contact.email.domain.name).toBe('UTpHKFyaaB');
+    expect(filtered.claim.contact.email.username).toBe('ZcMpCBQ0lE');
+  });
+
+  // TODO @jpsantosbh look at my merkle tree, my claim path there does not match my claims
```

Comment:
@dmelosantos @jpsantosbh Is the merkle tree correct? If so, let's remove this comment.